The Metal Pro Strings upgrade

kevinturner opened the pull request:
This is a very Guitar Hero-y pro guitar part, and would be so much fun if it weren't for the fingerstyle part.

kevinturner commented:
Here's a preview of the charts:
[![Alt text](https://img.youtube.com/vi/dsjC1Gg8XmM/0.jpg)](https://www.youtube.com/watch?v=dsjC1Gg8XmM)

jdk13 reviewed (changes requested):
I haven't run it in game myself but any reason for using custom chord name markers over the actual chord name sections in the bridge? I think the game would have outputted the right chord name you have for some of these (except for measure 37 I can see it outputting A7sus2 over Am7 since the 0 is the opening note you can keep that custom chord marker.) If you did try and it for reason didn't do that then I guess this is fine. rest of the chart is good

also try to give your difficulties real values instead of the default to avoid sorting alphabetically. (bass might be overtierred?)

kevinturner commented:
I used the provided custom chord names because I didn't care to fix them if they didn't work. However, I did notice in the process of investigating that two of the chord names (C6 & Am7) were being hidden when they really shouldn't be, so I went ahead and fixed that. Also adjusted the difficulties slightly down for bass and slightly up for guitar; bass should have always appeared one lower than what is in the video. I'm not sure what happened there, as it appears to be lower difficulty in Magma.

jdk13 reviewed (approved):
Okay in that case I'll allow it then. Unless before this gets merged someone else feels strongly against these markers being used in play of the real ones. But it's not a big enough deal to be considered like it's a huge malpractice (but this also doesn't mean anyone can use only event markers to name chords going for forward LOL)

this is technically ready to merge unless someone else strongly objects to the chord names in the bridge being event markers instead of real ones (except for measure 37)
